uint/src/uint.rs

```diff
@@ -385,6 +385,95 @@ pub fn split_u128(a: u128) -> (u64, u64) {
 	((a >> 64) as _, (a & 0xFFFFFFFFFFFFFFFF) as _)
 }
 
+#[inline(always)]
+#[doc(hidden)]
+pub fn div_mod_word(hi: u64, lo: u64, y: u64) -> (u64, u64) {
+    debug_assert!(hi < y);
```

Comment:
There's some funny white space for many of these `debug_assert`.

Reply:
yeah, also code duplication, will mark the PR as "ready for review" after fixing that